app/assets/javascripts/components/article_finder/article_finder.jsx

```diff
@@ -454,4 +459,7 @@ const mapDispatchToProps = {
   clearResults: clearResults,
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(ArticleFinder);
+export default compose(
```

Comment:
What is this change for?

Reply:
I added withRouter to access location, because I'm passing project if changed in AssignButton for ArticleFinder to render right version.

https://github.com/1v4n4/WikiEduDashboard/blob/6f593ae734a857ae97fd94c60a4fcf7c4af26019/app/assets/javascripts/components/common/AssignCell/AssignButton.jsx#L202

Reply:
I don't understand where `location` comes into it. It looks like the project comes through a prop.

Reply:
In Article finder project is determined based on home_wiki. 
My logic was to pass it through link_to from AssignButton, so I could work with something other than a home wiki. To access that prop in Article Finder with this.props.location.project I needed withRouter. Is this making any sense?

Reply:
Oh... is that because the prop is being passed through the LinkTo element?

Reply:
right

Reply:
Anyway. I just found another error to fix tomorro
![w](https://user-images.githubusercontent.com/65791349/147616600-05c6141e-65e5-4391-a3c5-a0de47be1cd9.png)
w